crates/anstyle-roff/src/roff_styles.rs

```diff
@@ -0,0 +1,43 @@
+#[derive(Debug, Clone)]
+pub(crate) enum Color {
+    Rgb(RgbColor),
+    AnsiColor(String),
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub(crate) struct RgbColor(pub u8, pub u8, pub u8);
+
+/// Constants for setting Control Requests to ROFF
+pub(crate) struct ControlRequests {}
+
+impl ControlRequests {
+    /// Control to Create a Color definition
+    pub const CREATE_COLOR: &'static str = "defcolor";
+    /// Roff control request to set background color (fill color)
+    pub const BACKGROUND: &'static str = "fcolor";
+    /// Roff control request to set foreground color (glyph color)
+    pub const FOREGROUND: &'static str = "gcolor";
+}
+
```

Comment:
Seems odd to make these associated constants rather than just putting them in the above mod or in an inner mod